### ISSUE TITLE
Axios auto add excess ‘localhost:3000/%E2%80%8B’ when requesting for an API by axios.get()!!!!

### ISSUE THREAD

**Issue by Anubhavdxt** [axios/axios#4830]
#### Describe the issue
On requesting for testnets-opensea API, Axios automatically adds a 'localhost:3000/%E2%80%8B', and I get an error *404 Not Found*. I've also tried with baseURL and all but the issue persists.

#### Example Code
Code snippet to illustrate your question

```js
// Example code here
  useEffect(() => {
    const getMyNfts = async () => {
      const openSeaData = await axios
        .get("​​https://testnets-api.opensea.io/assets?asset_contract_address=0x784Db7f90c42B75291733A2259e5b80Eb61DBCa6&order_direction=asc"
        );
      console.log(openSeaData.data.assets);
    };
    getMyNfts();
  }, []);
```

#### Expected behavior, if applicable
It should return an array of NFTs.

#### Environment
 - Axios Version: 0.27.2
 - Adapter: XHR/HTTP
 - Browser: Chrome
 - Browser Version: 103
 - Node.js Version: 16.13.1
 - OS: Windows 11
 - Additional Library: React 18.2

#### Additional context/Screenshots
Add any other context about the problem here. If applicable, add screenshots to help explain.
![image](https://user-images.githubusercontent.com/77921716/177079186-01f18500-9f0b-4603-98e6-6329895b4e0e.png)
![image](https://user-images.githubusercontent.com/77921716/177079248-f76c4081-2d8c-4703-9035-f379a080120c.png)

**Comment by Anubhavdxt** [comment]
> 

Yeah, I've tried this also but not getting any improvement issue persists.
It's giving the same error.

**Comment by BrunnerLivio** [comment]
@Anubhavdxt I believe I figured it out. You seem to have an invisible white space character. When I paste your code snippet into VSCode you can see it with the orange border. This invisible white space character correlates to `%E2%80%8B`. I guess you copy pasted the URL and something went wrong. 

I assume axios per default requests the current host in case your URL does not start with `http[s];//`. That is why it adds `localhost:3000`


![Unbenannt](https://user-images.githubusercontent.com/9899423/177122912-22af1a64-fce7-40ab-8b32-74b5ad628b68.PNG)

The following code snippet should fix it:

````javascript
useEffect(() => {
    const getMyNfts = async () => {
      const openSeaData = await axios
        .get("https://testnets-api.opensea.io/assets?asset_contract_address=0x784Db7f90c42B75291733A2259e5b80Eb61DBCa6&order_direction=asc"
        );
      console.log(openSeaData.data.assets);
    };
    getMyNfts();
  }, []);
```

Long story short; Use an editor which hints at invisible characters & be careful when copy/pasting :)

**Comment by Anubhavdxt** [comment]
Thank you so much @BrunnerLivio 
Really appreciate your efforts. I was badly stuck in it.
